src/logger.js

```diff
@@ -1,29 +1,68 @@
 import { TracerGlobals } from './globals';
-
+import { isStoreLogs } from './utils';
 const LOG_PREFIX = '#LUMIGO#';
 const WARN_CLIENT_PREFIX = 'Lumigo Warning';
 
+const MAX_DUPLICATE_LOGS = 50;
+
+export const LogStore = (() => {
+  let logSet = new Set([]);
+  let duplicateLogsCount = 0;
+
+  const addLog = (type, message, object) => {
+    const logObj = JSON.stringify({ type, message, object });
+    if (!logSet.has(logObj)) {
+      logSet.add(logObj);
+    } else {
+      duplicateLogsCount++;
```

Comment:
This means that we sum all duplicates instead of counting grouped by duplicate-type.
For example: `"a", "a", "b", "b"`  will be `4` instead of `{a: 2, b:2}`.
Is this the behavior we want?

Reply:
Yes its good enough for now